cmd/influx/remote.go

```diff
@@ -22,13 +23,60 @@ func newRemoteCmd() cli.Command {
 }
 
 func newRemoteCreateCmd() cli.Command {
+	var params remote.CreateParams
 	return cli.Command{
 		Name:   "create",
 		Usage:  "Create a new remote connection",
 		Before: middleware.WithBeforeFns(withCli(), withApi(true), middleware.NoArgs),
-		Flags:  commonFlags(),
-		Action: func(ctx *cli.Context) {
-			fmt.Println("remote create command was called")
+		Flags: append(
+			commonFlags(),
+			&cli.StringFlag{
+				Name:        "name, n",
+				Usage:       "Name for the new remote connection",
+				Required:    true,
+				Destination: &params.Name,
+			},
+			&cli.StringFlag{
+				Name:        "description, d",
+				Usage:       "Description for the new remote connection",
+				Destination: &params.Description,
+			},
+			&cli.StringFlag{
+				Name:        "org-id",
+				Usage:       "The ID of the local organization",
+				EnvVar:      "INFLUX_ORG_ID",
+				Destination: &params.OrgID,
+			},
+			&cli.StringFlag{
+				Name:        "remote-url",
+				Usage:       "The url for the remote database",
+				Required:    true,
+				Destination: &params.RemoteURL,
+			},
+			&cli.StringFlag{
+				Name:        "remote-api-token",
+				Usage:       "The API token for the remote database",
+				Destination: &params.RemoteAPIToken,
+			},
+			&cli.StringFlag{
+				Name:        "remote-org-id",
+				Usage:       "The ID of the remote organization",
+				Required:    true,
+				Destination: &params.RemoteOrgID,
+			},
+			&cli.BoolFlag{
+				Name:        "allow-insecure-tls",
+				Usage:       "Allows insecure TLS",
```

Comment:
Would add an explicit default of `false`

Reply:
I think the default is already false, by using `BoolFlag{}` rather than `BoolTFlag{}`? I don't see a value field for `BoolFlag{}`, similar to how you would set a default value for a string flag.

Reply:
Ah, then this is ok :+1: